src/components/ShellView/ResultTable.vue

```diff
@@ -256,4 +296,32 @@ export default {
     margin-bottom: 0;
   }
 }
+
+.data-container {
```

Comment:
You should clean-up your CSS:
1. Use BEM naming convention.
2. Use sass nested CSS structure when possible.
3. Remove commented-out code.